deployment/scripts/prometheus-deploy.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -o allexport && source .env && set +o allexport 
+
+if [ "${k8s_provider}" == "eks" ]; then
+    echo "Updating your kube context locally ...."
+    aws eks update-kubeconfig --name ${TF_VAR_eks_cluster_name}
+    echo "Deploying kube-prometheus helm chart to ${TF_VAR_eks_cluster_name} ...."
+    helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    helm repo update
+    helm upgrade kube-prometheus prometheus-community/kube-prometheus-stack \
```

Comment:
we should be versioning external dependencies like third-party helm charts. Otherwise, we may run into unpredictable behavior.